libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -201,6 +302,14 @@ def create_react_agent(
             - str: This is converted to a SystemMessage and added to the beginning of the list of messages in state["messages"].
             - Callable: This function should take in full graph state and the output is then passed to the language model.
             - Runnable: This runnable should take in full graph state and the output is then passed to the language model.
+        handle_unanswered_tool_calls: An optional parameter that defines how to handle AIMessages with tool calls that
+            do not have corresponding ToolMessages. This is useful to handle errors from LLM providers that require
+            AIMessages with tool calls to be followed by ToolMessages with corresponding tool call IDs.
+            If provided, the list of messages sent to the model will be modified according to the specified strategy:
+
+            - "remove_tool_calls": modify AIMessages to remove tool_calls that do not have corresponding ToolMessages
+            - "add_tool_messages": add a ToolMessage for each unanswered tool call, following each AIMessage with unanswered tool calls
```

Comment:
What's the content of those messages?

Reply:
good question -- currently it's an empty string, but i wonder if it needs to be parametrized somehow / or if there is a better default